app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="container-fluid p-2">
+  <div class="card">
+    <h3 class="card-title text-center"><%= @recipe.name %></h3>
+    <div class="card-body">
+      <span class="text-start">Preparation time:<%= @recipe.preparation_time %> hours</span>
+       <% if user_signed_in? && current_user == @recipe.user %>
+      <div class="d-flex justify-content-end form-check form-switch">
+        <input class="form-check-input" type="checkbox" role="switch" id="flexSwitchCheckDefault" <%= 'checked' if @recipe.public %> >
```

Comment:
Good job for adding this switch button 👍🏼 , but please it needs to be functional by updating recipe when toggeled as mentioned here in the requirements

**Recipe details:**

Should display a toggle button that allows for a recipe to be made public or private.
If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
If the user is the owner of the recipe, should lead to the form that allows the user to add new food.